frontend/public/index.html

```diff
@@ -2,42 +2,17 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <meta name="description" content="Web site created using create-react-app" />
```

Comment:
It is good SEO practice to have here something meaningful, but no big deal. LGTM